.circleci/config.yml

```diff
@@ -13,7 +13,7 @@ defaults: &defaults
   - checkout
   - run:
       name: Install dependencies
-      command: sudo pip install -r requirements.txt flake8 pytest
```

Comment:
Do we need (make any changes to) requirements.txt?

Reply:
Actually, I will delete the requirements.txt file.